test/distribution/test_default_quantile_method.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Third-party imports
+import numpy as np
+import mxnet as mx
+
+# First-party imports
+from gluonts.mx.distribution.gaussian import Gaussian
+from gluonts.mx.distribution.mixture import MixtureDistribution
+
+
+def test_quantile() -> None:
```

Comment:
I did this test because with the mixture of two identical Gaussians we know the quantiles of the mixture which should be the quantiles of the Gaussian components. However, the quantiles of the single gaussian are computed with the quantile method of the Gaussian distribution while the quantiles of the mixture are computed with the cdf bisection method. This is a sanity check that this method gives proper results (or at least that it does not fail).

I will try to add also the tests you suggested.